_data/documentation.yaml

```
@@ -0,0 +1,78 @@
+- header: Overview
+  pages:
+  - title: Language refrence (TSPL)
+    url: /documentation/tspl/
+    description: |
+      The Swift Programming Language book is the authoritative reference for Swift,
+      offering a guided tour, a comprehensive guide, and a formal reference of the language.
+  - title: API Design Guidelines
+    url: /documentation/api-design-guidelines/
+    description: |
+      Delivering a clear, consistent developer experience when writing Swift code is largely defined by the names and idioms that appear in APIs.
+      These design guidelines explain how to make sure that your code feels like a part of the larger Swift ecosystem
+  - title: Standard Library
+    url: /documentation/standard-library/
+    description: |
+      The Swift standard library defines a base layer of functionality for writing Swift programs.
+  - title: Core Libraries
+    url: /documentation/core-libraries/
+    description: |
+      The Swift Core Libraries project provides higher-level functionality than the Swift standard library.
+      These libraries provide powerful tools that developers can depend upon across all the platforms Swift supports.
+  - title: Package Manager
+    url: /documentation/package-manager/
+    description: |
+      The Swift Package Manager is a tool for managing the distribution and use of "packages" of Swift code.
+      It's integrated with the Swift build system to automate the process of downloading, compiling, and linking dependencies into target products.
+  - title: REPL &  Debugger
+    url: /documentation/lldb/
+    description: |
+      - The LLDB debugger provides a rich REPL as well as the debugging environment for the Swift Language.
+#----------------------------------------------------
```

```suggestion
  - title: About Swift
    url: /about/
    description: |
      - A summary of the core features of Swift, supported platforms, and open source projects.
#----------------------------------------------------
```

It seemed that the About Swift page fit better as part of the Overview section as opposed to Articles. It also seemed strange to rename the page something like "Swift Overview" since the entire section is called Overview. So, I suggest leaving the name About Swift and having it at the end of the Overview section.